components/gateway/internal/auditlog/service.go

```diff
@@ -35,10 +37,15 @@ func (sink *Sink) Log(request, response string, claims proxy.Claims) error {
 		Claims:   claims,
 	}
 
+	log.Printf("Attempting to register auditlog message %+v for processing to the queue (size=%d, capacity=%d)\n",
```

Comment:
I think the log after writing to the channel should be enough, so this one can be removed